cluster/gce/util.sh

```diff
@@ -1395,6 +1414,10 @@ function kube-down() {
         --region "${REGION}" \
         --quiet \
         "${MASTER_NAME}"
+      # We need to restart master's kubelet as it uses the external IP which destination has just been changed.
```

Comment:
As discussed offline we should verify:
* why exactly is it needed
* why we don't need to do it for all other nodes
* what other options do we have? restarting kubelet is pretty intrusive